### PR TITLE
Add badges and license

### PR DESCRIPTION
**Pull request by RobertHebel**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
+
+Copyright (c) 2020 Brainhub
```
Comment (piotr-s-brainhub) [inline LICENSE:3]:
```suggestion
Copyright (c) 2019-2020 Brainhub
```

Comment (piotr-s-brainhub) [reply LICENSE:3]: The first commit was made in 2019 but this repo was transfered to the `brainhubeu` organization in 2020 (as I remember).

@rkostrzewski what do you think?

File: README.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # sqrs
 
+[![Last commit](https://img.shields.io/github/last-commit/brainhubeu/sqrs.svg)](https://github.com/brainhubeu/sqrs/commits/master)
+[![license](https://img.shields.io/npm/l/@brainhubeu/sqrs.svg)](https://github.com/brainhubeu/sqrs/blob/master/LICENSE)
+[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](http://makeapullrequest.com)
+[![Renovate enabled](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://renovatebot.com/)
+
+[![Downloads](https://img.shields.io/npm/dm/@brainhubeu/sqrs?color=blue)](https://www.npmjs.com/package/@brainhubeu/sqrs)
+[![Activity](https://img.shields.io/github/commit-activity/m/brainhubeu/sqrs.svg)](https://img.shields.io/bundlephobia/min/@brainhubeu/sqrs?label=minified)
```
Comment (piotr-s-brainhub) [inline README.md:9]:
```suggestion
[![Activity](https://img.shields.io/github/commit-activity/m/brainhubeu/sqrs.svg)](https://github.com/brainhubeu/sqrs/commits/master)
```

File: LICENSE (hunk 1 above)
Comment (rkostrzewski) [reply LICENSE]: 👍

File: README.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
 # sqrs
 
+[![Last commit](https://img.shields.io/github/last-commit/brainhubeu/sqrs.svg)](https://github.com/brainhubeu/sqrs/commits/master)
```
Comment (piotr-s-brainhub) [inline README.md:3]: I miss the build passed branch.

Comment (piotr-s-brainhub) [reply README.md:3]: still, I cannot see it

Comment (RobertHebel) [reply README.md:4]: sry, now it' added

**Pull request merged**